runtime/pkg/logutil/handler.go

```diff
@@ -2,17 +2,26 @@ package logutil
 
 import (
 	"github.com/rilldata/rill/runtime/pkg/logbuffer"
+	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
 type BufferedZapCore struct {
 	fields []zapcore.Field
 	logs   *logbuffer.Buffer
+	enc    zapcore.Encoder
 }
 
 func NewBufferedZapCore(logs *logbuffer.Buffer) *BufferedZapCore {
+	encCfg := zap.NewDevelopmentEncoderConfig()
+	encCfg.NameKey = zapcore.OmitKey
+	encCfg.EncodeLevel = nil
+	encCfg.EncodeTime = nil
+	consoleEncoder := zapcore.NewConsoleEncoder(encCfg)
```

Comment:
Is there a reason not to use `zapcore.NewJSONEncoder()`? Since this is for a JSON payload and only the fields are output